saleor/graphql/schema.graphql

```diff
@@ -3441,6 +3444,8 @@ type Query {
   webhooks(filter: WebhookFilterInput, before: String, after: String, first: Int, last: Int): WebhookCountableConnection
   webhookEvents: [WebhookEvent]
   webhookSamplePayload(eventType: WebhookEventTypeEnum!): JSONString
+  warehouse(id: ID!): Warehouse
+  warehouses(before: String, after: String, first: Int, last: Int): WarehouseCountableConnection
```

Comment:
I assume that `filter` would be useful here

Reply:
Yep, for consistency with other top-level queries we shuold also have a `filter`.